src/App.tsx

```diff
@@ -35,61 +36,123 @@ export function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
+  visibleGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case 1:
```

Comment:
use enum here, for now it's not clear what these value means

Reply:
done!